localplanning.services/src/components/Analytics.astro

```diff
@@ -0,0 +1,13 @@
+---
+const isProduction = import.meta.env.MODE === "production";
```

Comment:
This is read from the build script - this will only be true when `pnpm build --mode production` is called.

Docs: https://docs.astro.build/en/reference/programmatic-reference/#mode